frontend/actions/businessCards.js

```diff
@@ -5,7 +5,7 @@ import serverAddress from '../constants/serverAddress';
 export const ADD_CARD = 'ADD_CARD';
 
 async function fetchWithTimeout(resource, options = {}) {
-    const { timeout = 10000 } = options;
+    const { timeout = 20000 } = options;
```

Comment:
Z czego wynika konieczność zwiększenia timeoutu?

Reply:
To akurat wynika z tego, że wysyłanie zdjęcia może trwać długo i zbyt krótki timeout może powodować fałszywy komunikat o błędzie, podczas gdy zdjęcie wciąż jest wysyłane na chmurę.